packages/html/src/toolbar/templates/toolbar-adaptive.tsx

```diff
@@ -0,0 +1,7 @@
+import ToolBar from "../toolbar.spec";
+
+export const ToolbarAdaptive = ({ children, ...other }) => (
```

Comment:
In my opinion, this template should be named ToolbarResizable instead as the only option it gets is _resizable_.